examples/README.md

```diff
@@ -4,12 +4,14 @@
 The following examples require you to create a file at `~/.ssh/tokens/snyk-api-token` which contains a valid Snyk API token - either your [personal token](https://app.snyk.io/account) or a [service account](https://snyk.io/docs/service-accounts/) token. This file should contain the token alone with no formatting.
 
 ## Running a Script
-The sample scripts are all a bit different, so you should try them each out or look at the code.
-
-In general the form is:
-```
-python examples/<script-name.py> --orgId=<your-org_id> ...
-```
+The sample scripts are all a bit different, so you should try them each out or look at the code. 
+
+1. The dependencies are managed by [poetry](https://poetry.eustace.io/). Installation is described on the [poetry docs](https://poetry.eustace.io/docs/#introduction)
+1. Install the dependencies using `poetry install`
+1. You can execute an example in general using:
+    ```
+    poetry run python examples/<script-name.py> --orgId=<your-org_id> ...
```

Comment:
Probably a good idea for now, but we should remember to remove this as soon as we have a package on PyPi. Poetry shouldn't bleed into the end user domain.

Reply:
@garethr I think we need instructions in the main README.md and possibly also in this README.md on how to install the dependencies with poetry.

Reply:
Poetry is't really intended as an end user tool. It would be good to have a `BUILDING .md` or `DEVELOP.md`I think to address the client developer need, (eg. https://github.com/docker/app/blob/master/BUILDING.md) but I think it sounds like we should separate the client and the examples to address the end user need best.

Reply:
Agreed. But first we need to publish it to pypi. And before we can publish, the minimal would be to get all examples in a working state.

Reply:
+1